python/helper-image/launcher/launcher.go

```diff
@@ -308,16 +309,23 @@ func (pc *pythonContext) updateCommandLine(ctx context.Context) error {
 		// Appropriate location to resolve pydevd is set in updateEnv
 		// TODO: check for modules (and fail?)
 		cmdline = append(cmdline, pc.args[0])
-		cmdline = append(cmdline, "-m", "pydevd", "--port", strconv.Itoa(int(pc.port)), "--server")
+		cmdline = append(cmdline, "-m", "pydevd", "--server", "--port", strconv.Itoa(int(pc.port)))
 		if pc.env["WRAPPER_VERBOSE"] != "" {
 			cmdline = append(cmdline, "--DEBUG")
 		}
-		if pc.debugMode == ModePydevdPycharm {
-			// From the pydevd source, PyCharm wants multiproc
-			cmdline = append(cmdline, "--multiproc")
+		if !pc.wait {
+			cmdline = append(cmdline, "--continue")
 		}
-		cmdline = append(cmdline, "--file") // --file is expected as last argument
-		cmdline = append(cmdline, pc.args[1:]...)
+
+		// --file is expected as last pydev argument, but it must be a file, and so launching with
+		// a module requires some special handling.
+		cmdline = append(cmdline, "--file")
```

Comment:
--file will always be there even when launching with a module, it seems.

To help me wrap my head around this, could you show a sample full command line invocation using modules? (e.g. pydevd [rest of the expected args])

Reply:
So `launcher --mode pydevd --port 5678 -- python -m flask` will end up creating a temp file:
```
$ cat /tmp/pydevd716531212/skaffold_pydevd_launch.py
import sys
import runpy
runpy.run_module('flask', run_name="__main__",alter_sys=True)
```
and will then invoke `python -m pydevd --server --port 5678 --DEBUG --continue --file /tmp/pydevd716531212/skaffold_pydevd_launch.py`

Reply:
ok perfect. thanks that helps a lot.

Reply:
maybe you should include that in comment form if any future reviewers have basic questions like this.

Reply:
Suggestions?  It does already say that `--file` is expected, and launching with a module requires special handling.  And the doc on handlePydevModule says what is done…

Reply:
you can ignore this if you want, but it helped me to see a concrete example as you showed here in the comments. will LGTM since you are planning on fixing that --file flag bug as a follow up

Reply:
Added a comment to the package doc comment at the top.